pkg/tester/data_perf.go

```diff
@@ -0,0 +1,93 @@
+// Copyright 2020 Coinbase, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package tester
+
+import (
+	"context"
+	"fmt"
+	"github.com/coinbase/rosetta-cli/configuration"
+	"github.com/coinbase/rosetta-sdk-go/fetcher"
+	"github.com/coinbase/rosetta-sdk-go/types"
+	"log"
+	"time"
+)
+
+func Setup_Benchmarking(config *configuration.Configuration) (*fetcher.Fetcher, func() time.Duration, chan time.Duration) {
+	// Create a new fetcher
+	fetcher := fetcher.New(
+		config.OnlineURL,
+		fetcher.WithRetryElapsedTime(time.Duration(config.RetryElapsedTime)*time.Second),
+		fetcher.WithTimeout(time.Duration(config.HTTPTimeout)*time.Second),
+		fetcher.WithMaxRetries(config.MaxRetries),
```

Comment:
if we retry, will we skew metrics? i.e if one block is failing, do we skew metrics by retrying

Reply:
I will set it to no retries